stanford-earth/SE3_BLT
# EARTH-15: Enable column text paragraph type.

# READY FOR REVIEW

# Summary
- Added column text paragraph type

# Needed By (Date)
- End of sprint

# Urgency
- low

# Steps to Test

1. Checkout this branch and same branch in [stanford_components](https://github.com/SU-SWS/stanford_components/pull/72) & [stanford_paragraph_types](https://github.com/SU-SWS/stanford_paragraph_types/pull/67)
2. config import
3. clear cache
4. create complex page with "Column Text" component.
5. change variants to check its display.

# Associated Issues and/or People
- https://github.com/SU-SWS/stanford_components/pull/72
- https://github.com/SU-SWS/stanford_paragraph_types/pull/67

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
